include/wx/webview.h

```diff
@@ -144,6 +144,27 @@ class WXDLLIMPEXP_WEBVIEW wxWebViewHandler
     wxString m_virtualHost;
 };
 
+class WXDLLIMPEXP_WEBVIEW wxWebViewConfigurationImpl
+{
+public:
+    virtual ~wxWebViewConfigurationImpl() = default;
+    virtual void* GetNativeConfiguration() const { return nullptr; }
+};
+
+class WXDLLIMPEXP_WEBVIEW wxWebViewConfiguration
```

Comment:
As written, this class shouldn't be DLL-exported as it only has inline functions. This would change, however, if `wxWebViewConfigurationImpl` becomes private as `GetNativeConfiguration()` couldn't be implemented inline then.